create.js

```diff
@@ -26,9 +26,12 @@ function create (modules, options) {
     var data = {}
     for (var i = 0, max = names.length; max > i; i++) {
       var name = names[i]
+      if (input[name] === 'false') {
+        input[name] = false
+      }
 
       // Directive attributes
-      if (name.indexOf(directive) === 0) {
+      if (name[0] === directive) {
```

Comment:
A directive could be more than one char which is why I went the indexOf route.  Other method could be to just hardcode the char instead of option.

Reply:
Oh yeah my bad, I missed that. this change didn't belong on this branch anyway